<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Search">
        <property name="text">
         <string>Поиск</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionSettings"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Language">
    <property name="title">
     <string>Язык</string>
    </property>
    <addaction name="actionRussian"/>
    <addaction name="actionEnglish"/>
    <addaction name="actionFinland"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>График</string>
    </property>
    <addaction name="actionDiagram1"/>
    <addaction name="actionChart1"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu"/>
   <addaction name="menu_Language"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSaveAs"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionAdd"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/img/icon-5b4ea.png</normaloff>:/rec/img/icon-5b4ea.png</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/img/save.png</normaloff>:/rec/img/save.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/img/settings.png</normaloff>:/rec/img/settings.png</iconset>
   </property>
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/img/export.png</normaloff>:/rec/img/export.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/img/delete-forever.png</normaloff>:/rec/img/delete-forever.png</iconset>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/img/Add.png</normaloff>:/rec/img/Add.png</iconset>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/img/save-as.png</normaloff>:/rec/img/save-as.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="actionRussian">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/img/Flag_of_Russia.svg.png</normaloff>:/rec/img/Flag_of_Russia.svg.png</iconset>
   </property>
   <property name="text">
    <string>Русский</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/img/flag_en_big.jpg</normaloff>:/rec/img/flag_en_big.jpg</iconset>
   </property>
   <property name="text">
    <string>Английский</string>
   </property>
  </action>
  <action name="actionFinland">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/img/finland.png</normaloff>:/rec/img/finland.png</iconset>
   </property>
   <property name="text">
    <string>Финский</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/img/ask-question.png</normaloff>:/rec/img/ask-question.png</iconset>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="actionDiagram1">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/img/graph.png</normaloff>:/rec/img/graph.png</iconset>
   </property>
   <property name="text">
    <string>Динамический график</string>
   </property>
  </action>
  <action name="actionChart1">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/img/klipartz.com.png</normaloff>:/rec/img/klipartz.com.png</iconset>
   </property>
   <property name="text">
    <string>Круговой график</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
